### PR TITLE
Fix postman mock server to serve data

### PR DESCRIPTION
**Pull request by krdurrett**
____ Wrote Tests __x__ Implemented ____ Reviewed

## Neccesary checkmarks:

    [] All Tests are Passing in all environments

    [x] The code will run locally

## Type of change

    [x] New feature
    [] Bug Fix
    [] Refactor

## Description of Change:

    description: Fixed postman mock server to serve data appropriately
    
## Requested feedback: 
    
    I'd like feedback on... n/a

## Check the correct boxes

    [x] This broke nothing
    [] This broke some stuff
    [] This broke everything

## Testing Changes

    [x] No Tests have been changed
    [] Some Tests have been changed
    [] All of the Tests have been changed(Please describe what in the world happened)

## Checklist:

    [] My code has no unused/commented out code
    [] My code has no binding.pry's
    [] I have reviewed my code
    [] I have commented my code, particularly in hard-to-understand areas
    [] I have fully tested my code

(For Fun!)Please Include a link to a gif of your feelings about this branch

Link:

### CODE REVIEW

**Review by jasmyn2244 — APPROVED** [review]
Big thanks to Evan! Let's start building!!!

File: src/Components/Home.js [hunk 1]
```diff
@@ -5,10 +5,10 @@ import { NavigationContext } from '../Context/NavigationContext'
 import { useQuery, gql } from "@apollo/client"
 
 const GET_CRYPTIDS = gql`
-    {
+    query GetCryptids {
```
Comment (RosaTheDev) [inline src/Components/Home.js:8]: I'll remember to add this for next time I use circle CI

**Review by RosaTheDev — APPROVED** [review]

File: src/Components/Home.js (hunk 1 above)
Comment (jasmyn2244) [reply src/Components/Home.js:8]: Me too

Comment (GraceGardner) [reply src/Components/Home.js:8]: ^^^

**Review by GraceGardner — APPROVED** [review]

**Pull request merged**